src/junit.rs

```diff
@@ -77,7 +82,36 @@ fn populate(
         failure_message: None,
         filename: rel_attrs.file,
         build_url: None,
-    })
+        computed_name: None,
+    };
+
+    match framework {
+        None => {
+            if let Some(f) = t.framework() {
```

Comment:
```suggestion
let framework = framework.or_else(|| t.framework());`
if let Some(f) = framework {
```

If I see things correctly, both match arms are the same, so this can deduplicate things.